### ISSUE TITLE
add wildcard support

### ISSUE THREAD

**Issue by dimaslanjaka** [isaacs/rimraf#256]
please add wildcard deleter.

bash example
```bash
find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +
```

rimraf
```bash
npx rimraf **/node_modules
```

**Comment by alfeg** [comment]
Question is why this were removed? It's basically what I've been using `rimraf` for - recursevly deleting files inside folder, i.e. `.orig` files and etc.

**Comment by isaacs** [comment]
@alfeg The recursive nature is still there.  Just the wildcard stuff was removed in v4.

It'll come back soon though.

dupe #249

**Comment by isaacs** [comment]
Landed on 4.2.0
